Build Failure when uncommenting user_config.h[#define DISABLE_STARTUP_BANNER]

### Expected behavior
Firmware build completes successfully with startup banner disabled.

### Actual behavior
Uncommenting user_config.h:#define DISABLE_STARTUP_BANNER causes a NodeMCU firmware build failure. Simply restoring the comment produces a successful build.

user_config.h:
=-=-=-=
.
.
.
// At start-up firmware details like:
//
// NodeMCU 3.0.1.0
//         branch:
//         commit:
//         release:
//         release DTS:
//         SSL: false
//         build type: integer
//         LFS: 0x0
//         modules: file,gpio,net,node,rtctime,sntp,tmr,uart,wifi
//  build 2020-01-27 17:39 powered by Lua 5.1.4 on SDK 3.0.2(824dc80)
//
// will be printed to serial console.  While it's mandatory for bug reports
// and good for development, it may be unwanted for non-interactive serial
// devices.

#define DISABLE_STARTUP_BANNER
.
.
.
=-=-=-=

[Windows 10 Pro (10.0.18363), Docker for Windows (2.2.0.5)]

docker run --rm -it -e "IMAGE_NAME=test1" -v "//c/Users/Joe/Google Drive/test1/nodemcu-firmware":/opt/nodemcu-firmware marcelstoer/nodemcu-build build
.
.
.
CC app/lua/lua.c
In file included from lua.c:11:0:
../lua53/lua.c:117:13: error: 'print_version' defined but not used [-Werror=unused-function]
 static void print_version (lua_State *L) {
             ^
cc1: all warnings being treated as errors
../../Makefile:422: recipe for target '.output/eagle/debug/obj/lua.o' failed
make[2]: *** [.output/eagle/debug/obj/lua.o] Error 1
make[2]: Leaving directory '/opt/nodemcu-firmware/app/lua'
../Makefile:380: recipe for target '.subdirs' failed
make[1]: *** [.subdirs] Error 2
make[1]: Leaving directory '/opt/nodemcu-firmware/app'
Makefile:380: recipe for target '.subdirs' failed
make: *** [.subdirs] Error 2
make: Leaving directory '/opt/nodemcu-firmware'

### Test code
n/a

### NodeMCU startup banner
NodeMCU 3.0.0.0 built with Docker provided by frightanic.com
	branch: dev
	commit: 88a33afd6e750d9be3d9e47ba7b3920cd73829da
	release: 2.0.0-master_20170202 +509
	release DTS: 202005011052
	SSL: false
	build type: float
	LFS: 0x10000 bytes total capacity
	modules: bit,file,gpio,i2c,net,node,tmr,uart,wifi
 build 2020-05-08 08:37 powered by Lua 5.1.4 on SDK 3.0.1-dev(fce080e)

### Hardware
Amica ESP8266 12E Development Board

--- a/app/lua53/lua.c
+++ b/app/lua53/lua.c
@@ -115,8 +115,10 @@ static int docall (lua_State *L, int narg, int nres) {
 }
 
 static void print_version (lua_State *L) {
+  #ifndef DISABLE_STARTUP_BANNER
   lua_writestringerror( "\n" NODE_VERSION " build " BUILD_DATE
                         " powered by " LUA_RELEASE " on SDK %s\n", SDK_VERSION);
+  #endif
 }
 
 
@@ -239,9 +241,7 @@ static int pmain (lua_State *L) {
   input_setup(LUA_MAXINPUT, get_prompt(L, 1));
   lua_input_string(" \n", 2);               /* queue CR to issue first prompt */
 
-#ifndef DISABLE_STARTUP_BANNER
   print_version(L);
-#endif
  /*
   * And last of all, kick off application initialisation.  Note that if
   * LUA_INIT_STRING is a file reference and the file system is uninitialised

